src/twilio_util.ts

```diff
@@ -37,25 +37,23 @@ export type TwilioRequestBody = {
   [mediaKey: string]: string | undefined;
 };
 
-export function formatAttachments(reqBody: TwilioRequestBody): string {
+/** Returns list of URLs for MMS attachments */
+export function getAttachments(reqBody: TwilioRequestBody): string[] {
   const numMedia = Number(reqBody.NumMedia);
 
   if (numMedia === 0) {
```

Comment:
since this is going to run for every single received message, can we only do the Number() conversion if !== 0? In other words, check numMedia === "0" as a string so that we can skip the conversion for 99.9% of messages